client/src/redux/slices/urls.js

```diff
@@ -0,0 +1,45 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import getUrlsQuery from '../../graphQl/queries/getUrlsQuery';
+import axios from '../../utils/axios';
+
+import { FETCH_URLS, SET_SELECTED_URL } from '../actionTypes';
+
+export const fetchUrls = createAsyncThunk(FETCH_URLS,
+    async (payload) => {
+      try {
+        const { token } = payload;
+        const graphqlQuery = getUrlsQuery();
+        const response = await axios.post('/', graphqlQuery, {
+          headers: {
+            Authorization: token,
+          },
+        });
+        const urls = response.data.data.getUrls.urls;
+
+        return { urls };
+      } catch (err) {
+        return { urls: [] };
+      }
```

Comment:
What is the cause of this error? Does setting the list of URLs to an empty array is the proper way of handling this error?